src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchBox.scss';
+
+class SearchBox extends Component {
+  constructor(props) {
+    super();
```

Comment:
Also, this is moot since we're removing constructor but you should always pass props to super()